mrp_production_partner_note/models/procurement_order.py

```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _prepare_mo_vals(self, procurement):
+        res = super(ProcurementOrder, self)._prepare_mo_vals(procurement)
+        sale_proc = procurement.move_dest_id.procurement_id
+        mrp_notes = sale_proc.sale_line_id.order_id.partner_id.mrp_notes
+        if mrp_notes:
+            res['notes'] = (
+                res.get('notes') and
+                ('{other}\n'.format(other=res.get('notes', ''))) or '')
+            res['notes'] += '{mine}'.format(mine=mrp_notes)
```

Comment:
I commented above but I put you here:

I am so sorry you are right.
 the `+=` sum also using numeric elemente.

The correct approach.
```python
    old = res.get('notes') and ('{other}\n'.format(other=res.get('notes', '')) or ''
    res['notes'] = '{old}{mine}'.format(old=old, mine=mrp_notes)
```